docs/source/privacy.mdx

```diff
@@ -1,24 +1,24 @@
 ---
 title: Privacy and Data Collection
-subtitle: Learn what data the Apollo Router collects and how to opt out
-description: Learn what data the Apollo Router collects and how to opt out. By default, the Apollo Router collects anonymous usage data to help improve the product.
+subtitle: Learn what data the GraphOS Router collects and how to opt out
```

Comment:
Available in both Router Core and GraphOS Router (for all occurrences on this page)